proposals/2021-04-lower-deposit-requirement/proposal.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Parameter change: lower minimum proposal deposit amount",
+  "description": "The current deposit amount of 512 ATOMs prohibits valuable governance activity from small holders or those with most of their ATOM staked. We propose lowering the requirment to 64 ATOMS.",
+  "changes": [
+    {
+      "subspace": "mindeposit",
+      "key":"depositparams",
+      "value": "64000000"
```

Comment:
```suggestion
      "subspace": "gov",
      "key":"depositparams",
      "value": {
            "mindeposit": [
            {
                  "denom": "uatom",
                  "amount": "64000000"
            }
            ]
}
```

Reply:
not sure if it resets the other deposit params tho...

Reply:
Ok, so worst case the update will fail

Reply:
cool, this looks right based on the issue you linked. gov uses different format, bizarre. will maybe wait to get another thumbs up before committing.

Reply:
This is not fully correct. I wrote docs on this: https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/spec/06_params.md

The key should be `min_deposit`, not `mindeposit`. JSON keys are snake-cased.

Also, per the doc above, you do not need to include the entire params JSON object, just the subset you want to change 👍

Reply:
is `gov` the correct subspace?

Reply:
Yup!

Reply:
ok pushed this change